site/contents/write_ups/2021/11-08-21_writeup.md

```diff
@@ -0,0 +1,119 @@
+
+A similar issue applies to technology. Ironically, hardware and software people are quite separate and as the majority of the data ethics club are more software focussed, a lot of us were surprised to learn how little we knew about where the hardware came from. Do hardware experts know any better?
+
+Do we need to regulate this somehow? If we are better at tracking this with data science tools, would we be better able to regulate it? If we had that kind of information, would that lead consumers to change anything? If we leave the market to fix things, will it fix it? Likely not!
+
+It should be the responsibility of companies to share data on the environmental impacts of their work is. There is currently very little in terms of insentives for companies to share this information. Getting companies to do this, whether through policy (stick) or other insentives (carrot) is the first step. The carrot to stick ratio is another debate to be had, but from what we've seen in the past, if the only thing stopping companies acting in bad faith for profit is good will, companies will tend to prioritise profits over the environment.
+
+From there it should be our responsibility as consumers to decide which companies to buy from - in theory. 
+
+This is not just an issue for big cooperations however. We all live on this planet, so we all need to be concerned about climate change. Even if we're fortunate to be somewhere less affected by climate change, the fact that we have can afford to make choices that are more environmentally friendly is a privedlige, one that we should not squander.
+
+That being said, the actions of some will make more of a difference than others. The classic: "Everyone stop rinsing dishes before putting dishes in the dishwasher because that'll stop climate change" is a nice idea but fails to acknowledge the comparative cost of [launching billionaires into space](https://www.independent.co.uk/climate-change/news/bezos-rocket-carbon-emissions-blue-origin-b1888059.html). Grassroots change and influence will matter, but that is likely to focus on small singular issues. Focus on the small may not get us where we need to be, nor nearly quick enough.
+
+Fixing climate change is a bit like a terrible group project. Seeing larger contributers slack off can make it tempting to throw our hands up in defeat and resign ourselves to our fate. This same cynicism is seen for example among people who choose not to vote, asking "what difference can one person make?" Of course every vote counts (even though in some systems, some votes count more than others), but it leads to a question of what can any single person do about it?
+
```

Comment:
Something Charles mentioned that seems important to fit in somewhere - maybe here?
```suggestion
Lastly, we also need to remember that not all of us have the same choices to act differently. The choice of which iPhone to buy next is a marker of a privilege that not everyone has.
```